meta-mel/fsl-arm/recipes-core/images/console-image.bbappend

```diff
@@ -0,0 +1 @@
+IMAGE_INSTALL += " u-boot"
```

Comment:
Why would we want u-boot installed in the rootfs? Does the early stage bootloader know how to load it from there? This isn't generally how things are done, in my experience.